.env.example

```diff
@@ -65,7 +65,6 @@ LEMON_SQUEEZY_PATH=
 GITHUB_PERSONAL_ACCESS_TOKEN=
 GITHUB_CLIENT_ID=
 GITHUB_CLIENT_SECRET=
-GITHUB_REDIRECT_URI=
```

Comment:
Given that there are now two different callbacks, the redirect URL is not loaded from the config, it is set up dynamically in each of the redirecting endpoints